DropInDemo/Demo Base/Settings/BraintreeDemoSettings.h

```diff
@@ -25,6 +30,7 @@ typedef NS_ENUM(NSInteger, BraintreeDemoTransactionServiceThreeDSecureRequiredSt
 + (NSString *)authorizationOverride;
 + (BOOL)useTokenizationKey;
 + (BraintreeDemoTransactionServiceThreeDSecureRequiredStatus)threeDSecureRequiredStatus;
++ (BraintreeDemoTransactionServiceThreeDSecureRequestedVersion)threeDSecureRequiredVersion;
```

Comment:
Shouldn't this be `threeDSecureRequestedVersion`? _Requested_ rather than _required_.